app/_src/policies/introduction.md

```diff
@@ -1,6 +1,651 @@
 ---
 title: Policies
 ---
+{% if_version gte:2.9.x %}
+## What is a policy?
+
+A policy is a set of configuration that will be used to generate the proxy configuration.
+{{ site.mesh_product_name }} combines policies with dataplane configuration to generate the Envoy configuration of a proxy.
+
+## What do policies look like?
+
+Like all [resources](/docs/{{ page.version }}/introduction/concepts#resource) in {{ site.mesh_product_name }}, there are two parts to a policy:
+
+1. The metadata
```

Comment:
imo this list while reading is very bold and grabs your eye for relatively little information content. I think it can be an inline sentence and then we have the sections.